### ISSUE TITLE
Delexicalized version for parser

### ISSUE THREAD

**Issue by tiberiu44** [adobe/NLP-Cube#25]
Small treebanks for UD are not suitable for our model. As such, we should train delexicalized versions of the parser, by combining compatible treebanks for those languages. Also, we could train delexicalized parsers directly on the small treebanks because using only morphology should not yield data sparseness on small datasets.

## Expected Result

We need to add another flag in the parser's config file for using lexical features or not: USE_LEXICAL
This should be similar to USE_MORPHOLOGY

## Details for implementation
`io_utils/config.py` <- we need to add the attribute here. We also need to be **backwards compatible** with older config files which don't have this flag enabled. Default value should be `True`
`generic_networks/parsers.py`
- `__init__(self...)` should use this information when computing the input (lines 45--62)
- `make_input(self,..)` should use this information to create the input: I recommend to branch the execution before line 260. I don't think dropout or any other mechanism will help prevent overfitting, so right now we should not be doing it. Just go through each entry, lookup Morphological information and sum-up the vectors (`t_emb = upos_emb + xpos_emb + attrs_emb`), than add to the sequence `x_list.append(w_emb)`

## For PR to pass it should:
- Be **able to train** on large datasets and provide results same as before the update (`USE_LEXICAL=True`, `USE_MORPHOLOGY=False`, `PREDICT_MORPHOLOGY=True`)
- Be **able to load pretrained models**, which don't have the `USE_LEXICAL` flag at all
- Be **able to train and load delexicalized models** for small treebanks.
